### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by schalkneethling**
Minor update to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# mdn-community
+# mdn-communtiy
 
-We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors. The MDN Web Docs project welcomes contributions from everyone who shares our goals and wants to contribute in a healthy and constructive manner within our community.
+We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors. The MDN Web Docs project welcomes contributions from everyone who shares our goals and wants to contribute healthily and constructively to our community.
```
Comment (dipikabh) [inline README.md:3]: How about something along the lines of: 
```suggestion
Welcome to the MDN Web Docs project! We are delighted with your interest in this project.

At MDN, we put people first and do our best to recognize, appreciate, and respect the diversity of our global contributors. We welcome contributions from everyone who shares our goals and wants to contribute healthily, respectfully, and constructively to this project.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
-MDN community is the repository where you will find the [offical MDN Web Docs public feedback discussions](https://github.com/mdn/mdn-community/discussions).
+MDN community is the repository where you will find the official MDN Web Docs [public feedback discussions](https://github.com/mdn/mdn-community/discussions). By participating in and contributing to, our projects and discussions, you acknowledge that you have read and agree to the [Mozilla community participation guidelines](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md).
```
Comment (dipikabh) [inline README.md:5]:
```suggestion
`mdn-community` is the repository where you will find the [public feedback discussions](https://github.com/mdn/mdn-community/discussions) for the MDN Web Docs project.

> **Note:** By participating in and contributing to our projects and discussions, you acknowledge that you have read and agree to the [Mozilla community participation guidelines](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md).
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for your work on updating this page, @schalkneethling! Leaving a few suggestions for your consideration. Let me know what you think.

**Comment by schalkneethling** [comment]
Thanks, @dipikabh, committed your suggestions.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
-# mdn-community
+# mdn-communtiy
```
Comment (dipikabh) [inline README.md:1]:
```suggestion
# mdn-community
```

**Review by dipikabh — APPROVED** [review]

**Pull request merged**